backend/src/model/dto.rs

```diff
@@ -167,7 +169,6 @@ pub struct PageParameters {
 #[aliases(
     PagePlantsSummaryDto = Page<PlantsSummaryDto>,
     PageSeedDto = Page<SeedDto>,
-    PageMapDto = Page<MapDto>,
```

Comment:
Why is this removed? It is still used in `map.rs` (although it is missing as a schema in `api_doc.rs`).